adafruit_connection_manager.py

```diff
@@ -184,54 +183,75 @@ def __init__(
     ) -> None:
         self._socket_pool = socket_pool
         # Hang onto open sockets so that we can reuse them.
-        self._available_socket = {}
-        self._open_sockets = {}
+        self._available_sockets = set()
+        self._managed_socket_by_key = {}
+        self._managed_socket_by_socket = {}
```

Comment:
This dict stores keys, not managed sockets, so call it `_key_by_managed_socket`, I think.

Reply:
@dhalbert fixed here: f33d78c